README.md

```diff
@@ -44,3 +44,15 @@ Linters order above is the preffered way to run and fix them one by one.
     or `set ...` for Windows.
 3. Type `flask run` command in terminal.
 4. Open http://127.0.0.1:5000 in your internet browser.
+
+
+
+### Run database container
+
+1. In terminal type in `docker-compose -f envs/dev/docker-compose.yml up --detach` command. (you need Docker and Docker Compose installed)
+
+
+
+### Apply migrations to database
```

Comment:
Let's name this section "Migrations" or "Working with migrations" and add more info to it. Let's add info about how to:
- apply migrations (`upgrade`)
- revert migrations (`downgrade`)
- autogenerate migrations